### PR TITLE
DM-2: Motor module implementation

### PR DESCRIPTION
**Pull request by scamper07**
- Added motor class with methods to turn motor clockwise and anti clockwise

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (Vatican-Cameos) [inline .vscode/settings.json]: Is this vscode directory required to be on the repo ? Better to remove IDE specific stuff. @scamper07

File: src/utils.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import os
+
+
+def setup_env():
+    # replace RPi.GPIO with fake GPIO if environment is DEV
+    if os.getenv("ENV") != "PROD":
+        print("Running in dev mode")
```
Comment (Vatican-Cameos) [inline src/utils.py]: Can you merge Logger change to your branch and change this print to a log statement instead ?

File: .vscode/settings.json (hunk 1 above)
Comment (scamper07) [reply .vscode/settings.json]: Removed vscode folder from git

File: src/utils.py (hunk 2 above)
Comment (scamper07) [reply src/utils.py]: Merged

**Review by Vatican-Cameos — APPROVED** [review]

**Pull request merged**